src/setupTest.js

```diff
@@ -1,2 +1,80 @@
 import 'core-js/stable';
 import 'regenerator-runtime/runtime';
+
+jest.mock('@edx/frontend-platform/i18n', () => {
+  const i18n = jest.requireActual('@edx/frontend-platform/i18n');
+  const PropTypes = jest.requireActual('prop-types');
+  const { formatMessage } = jest.requireActual('./testUtils');
+  const formatDate = jest.fn(date => new Date(date).toLocaleDateString()).mockName('useIntl.formatDate');
+  return {
+    ...i18n,
+    intlShape: PropTypes.shape({
```

Comment:
we don't need to mock these (`intlShape`, `useIntl`) for this repo as they will not be used.